src/ert/storage/local_ensemble.py

```diff
@@ -487,6 +673,12 @@ def save_parameters(
 
         dataset.expand_dims(realizations=[realization]).to_netcdf(path, engine="scipy")
 
+        if os.path.exists(self._path / f"{group}.nc"):
```

Comment:
How likely is this, and is there any way we can guard against it ever happening?

Reply:
Not sure, we could just raise an error if this is done. That would enforce that we never call `save_parameters` after doing `load_parameters` and likewise for responses.